alacritty/src/event.rs

```diff
@@ -1179,8 +1179,11 @@ impl input::Processor<EventProxy, ActionContext<'_, Notifier, EventProxy>> {
                         // Minimizing the window sends a Resize event with zero width and
                         // height. But there's no need to ever actually resize to this.
                         // ConPTY has issues when resizing down to zero size and back.
+                        //
+                        // The user could also resize the window to have zero height
+                        // due to the titlebar.
                         #[cfg(windows)]
-                        if size.width == 0 && size.height == 0 {
+                        if size.width == 0 || size.height == 0 {
```

Comment:
Is this actually creating ConPTY issues, or is our code struggling with the zero size? You mentioned winit, so if that's a change introduced to winit we might have to add this check to all platforms?

Reply:
I mean you can't resize the window to the size of zero, you can't have a buffer that way? Our code will have issues with zero sizes, because you can't make a zero buffer, but that's nothing we can do other than ignoring...

Reply:
Note, that it's probably wise to ignore zero sizes on all platforms, just in case, but yet I haven't seen them coming anywhere other than windows...

Reply:
> I mean you can't resize the window to the size of zero, you can't have a buffer that way? Our code will have issues with zero sizes, because you can't make a zero buffer, but that's nothing we can do other than ignoring...

Can you not do that on other platforms though? I'm pretty sure I added an i3 option to limit my window size because I can resize it to 0x0.

Reply:
Ah it seems like X11 backend might be clamped to 1x1 internally. I can make the window completely disappear but it still reports a resize event of 1x1.